src/List/ListItemText.js

```diff
@@ -14,6 +14,9 @@ export const styleSheet = createStyleSheet('ListItemText', () => {
         paddingLeft: 0,
       },
     },
+    inset: {
+      paddingLeft: '56px !important',
```

Comment:
The best solution I can think of is to increase the specificity with `'&:first-child': {`.